lib/util/version.js

```diff
@@ -14,8 +14,19 @@ function getFromContext(context) {
   return confVer.split('.').map(part => Number(part));
 }
 
-function test(context, methodVer) {
-  const confVer = getFromContext(context);
+function getFlowVersionFromContext(context) {
+  let confVer = '999.999.999';
+  // .eslintrc shared settings (http://eslint.org/docs/user-guide/configuring#adding-shared-settings)
+  if (context.settings.react && context.settings.react.flowVersion) {
+    confVer = context.settings.react.flowVersion;
+  } else {
+    throw 'Could not retrieve flowVersion from settings';
```

Comment:
Sorry if I wasn't clear; when no version is specified, I want the logic to be "guess which flowVersion it is" - specifying a flow version can't be required, or else it's a very large breaking change.